storage-bigtable/src/utils.rs

```diff
@@ -0,0 +1,291 @@
+use solana_sdk::{
```

Comment:
Generic file/directory names like "utils" are pretty triggering for me.   Maybe this can be called `from.rs`?  `convert.rs`?

Reply:
Way better. Thanks!